scripts/litePageSizeValidator/index.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+import { exec } from 'node:child_process';
+
+const litePageSizeValidator = async () => {
+  const urlsToCheck = [
+    '/hindi',
+    '/mundo/articles/cddylv9g8z0o',
+    '/nepali/bbc_nepali_radio/liveradio',
+    '/arabic/media-53135426',
+    '/marathi/popular/read',
+    '/gahuza/bbc_gahuza_radio/programmes/p0340x2m',
+    '/nepali/news-50627370',
+    '/arabic/sports-54278377',
+    '/korean/topics/cnwng7v0e54t',
+  ];
+
+  const testResults = await Promise.all(
+    urlsToCheck.map(async url => {
+      const pageSize = exec(
```

Comment:
Does this need to be awaited? 
```suggestion
      const pageSize = await exec(
```

Reply:
OR does the use of Promise.all do away with the need for async/await (and if so means we can remove them?)